notebooks/demo_generate_prsn.ipynb

```diff
@@ -0,0 +1,396 @@
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Dry Run\n",
+      "File: /tmp/pywps_process_lagkiopl/pr_week_test.nc\n",
+      "project: CMIP5\n",
+      "model: NorESM1-M\n",
+      "institute: PCIC\n",
+      "experiment: historical,rcp26\n",
+      "ensemble_member: r1i1p1\n",
+      "dependent_varnames: ['pr']\n",
+      "File: /tmp/pywps_process_lagkiopl/tasmin_week_test.nc\n",
+      "project: CMIP5\n",
+      "model: NorESM1-M\n",
+      "institute: PCIC\n",
+      "experiment: historical,rcp26\n",
+      "ensemble_member: r1i1p1\n",
+      "dependent_varnames: ['tasmin']\n",
+      "File: /tmp/pywps_process_lagkiopl/tasmax_week_test.nc\n",
+      "project: CMIP5\n",
+      "model: NorESM1-M\n",
+      "institute: PCIC\n",
+      "experiment: historical,rcp26\n",
+      "ensemble_member: r1i1p1\n",
+      "dependent_varnames: ['tasmax']\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Process dry run output\n",
+    "req = requests.get(dry_output.get()[0])\n",
+    "print(req.content.decode('utf-8'))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 19,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "application/vnd.jupyter.widget-view+json": {
+       "model_id": "ccd84fef119346659b9f39d71f6adab4",
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/plain": [
+       "HBox(children=(IntProgress(value=0, bar_style='info', description='Processing:'), Button(button_style='danger'…"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# generate prsn\n",
+    "output = thunderbird.generate_prsn(pr_file_local, tasmin_file_local, tasmax_file_local, chunk_size=50, dry_run=False)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 20,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "<?xml version=\"1.0\" encoding=\"UTF-8\"?><html><body><metalink xmlns=\"urn:ietf:params:xml:ns:metalink\">\n",
+      "<published>2020-06-05T15:46:15Z</published>\n",
+      "<generator>PyWPS/4.2.2</generator>\n",
+      "<file name=\"prsn_day_BCCAQv2_NorESM1-M_historical+rcp26_r1i1p1_19500101-19500107_Canada.nc\">\n",
+      "<identity>prsn_day_BCCAQv2_NorESM1-M_historical+rcp26_r1i1p1_19500101-19500107_Canada.nc</identity>\n",
+      "<size>7665520</size>\n",
+      "<metaurl mediatype=\"application/x-netcdf\">http://localhost:5001/outputs/5cf2af8b-a77e-11ea-9436-c86000e3f2fd/prsn_day_BCCAQv2_NorESM1-M_historical+rcp26_r1i1p1_19500101-19500107_Canada.nc</metaurl>\n",
+      "<publisher name=\"None\" url=\"http://localhost:5001/wps\"></publisher>\n",
+      "</file>\n",
+      "</metalink></body></html>\n"
+     ]
+    }
+   ],
+   "source": [
+    "from bs4 import BeautifulSoup\n",
+    "req = requests.get(output.get()[0])\n",
+    "print(BeautifulSoup(req.content.decode('utf-8')))"
```

Comment:
Could you clarify what you mean by slightly broken?

Reply:
I'm not sure if you are experiencing this same output or if it is just my machine. The output on my local machine comes out to: 
```
<metaurl mediatype="application/x-netcdf">http://localhost:5001/outputs/600d0f27-a77e-11ea-9436-c86000e3f2fd/prsn_test.nc&lt;/metaurl&gt;
```
I'm not sure what is causing the `&lt;/metaurl&gt` portion but that is what I have to remove to use the link. It would be nice to have the final output just be a clickable link that doesn't need any manual intervention to navigate to.

Reply:
Ah. That's what I thought (I'm getting the same thing).

Reply:
You probably know this already, but the weird encoding is HTML: e.g., `&lt;` is HTML encoding for `<`. Not sure if this will help you.